app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -47,13 +50,13 @@ params:
       default:
       value_in_examples:
       description: |
-        One of more request per window to apply
+        One or more requests-per-window limits to apply.
     - name: window_size
       required:
       default:
       value_in_examples:
       description: |
-        One more more window sizes to apply (defined in seconds)
+        One or more window sizes to apply limit to (defined in seconds).
```

Comment:
I'd suggest either saying "apply a limit to" or "apply `limit` to"

Reply:
> I'd suggest either saying "apply a limit to" or "apply `limit` to"

went with "apply a limit to"